### PR TITLE
Bugfix - Change output of login from stderr to stdout

### PR DESCRIPTION
**Pull request by naomi-friedrich**
In the past, when you ran `runai login` it would have print output of "Username: " and "Password: " and send it to `Stderr`, instead of `Stdout` which is the correct way in linux standard.

We have found this issue through automation process

### CODE REVIEW

**Comment by itayvallach** [comment]
😂😂😂😂

File: pkg/authentication/flows/password/password.go [hunk 1]
```diff
@@ -133,7 +133,7 @@ func getRawCredentials() (string, string, error) {
 }
 
 func readString(prompt string) (string, error) {
-	if _, err := fmt.Fprint(os.Stderr, prompt); err != nil {
+	if _, err := fmt.Fprint(os.Stdout, prompt); err != nil {
```
Comment (omer-dayan) [inline pkg/authentication/flows/password/password.go:136]: Dont you think we need to refactor all the logic?

Comment (omer-dayan) [reply pkg/authentication/flows/password/password.go:136]: Did you think about utilization?

Comment (omric-runai) [reply pkg/authentication/flows/password/password.go:136]: I think we should consider adding about 3-4 utilizations in here

**Comment by enoodle** [comment]
Thank you for your contribution @naomi-friedrich

File: pkg/authentication/flows/password/password.go (hunk 1 above)
Comment (oferla) [reply pkg/authentication/flows/password/password.go:136]: need to check if os.Stdout is not nil.

**Review by oferla — APPROVED** [review]

**Review by natasharomm — APPROVED** [review]

**Pull request merged**